packages/rancher-cis-benchmark/charts/Chart.yaml

```diff
@@ -1,9 +1,10 @@
 apiVersion: v1
-appVersion: v0.0.1
+appVersion: v1.0.0
 description: The cis-operator enables running CIS benchmark security scans on a kubernetes
   cluster
 name: rancher-cis-benchmark
-version: 0.0.1
+version: 1.0.000
+icon: https://charts.rancher.io/assets/logos/cis-kube-bench.svg
```

Comment:
This doesn't actually exist

Reply:
Seems like all the logo images are in dev-2.5 branch - so I added it to dev-2.5 PR

Reply:
Is this the way we need to follow? or there is some other method to get this added? @cbron

Reply:
Whoops, missed your message above. Yea that works, so this PR is good.